www_src/pages/element/link-editor.jsx

```diff
@@ -13,8 +13,20 @@ var LinkEditor = React.createClass({
   getInitialState: function () {
     return LinkBlock.spec.flatten(this.props.element, {defaults: true});
   },
-  componentDidUpdate: function () {
-    this.props.cacheEdits(this.state);
+  componentDidUpdate: function (prevProps, prevState) {
+    var state = this.state;
+
+    // Update state if parent properties change
+    if (this.props !== prevProps) {
```

Comment:
it actually looks like React passes the reference this.props if prevProps hasn't been updated